src/asciidoc/docs/publish/customDomains.adoc

```diff
@@ -0,0 +1,84 @@
+= Using custom domains
+:page-slug: custom-domains
+:page-description: How to configure your Koji to use a custom domain.
+
+When you're ready to go live with your template, you might want to use a custom domain name in place of the name provided by Koji.
+A custom domain can have the advantage of being easier to remember, and it usually just looks better.
+
+Koji has a few subdomains that you can use at no cost and with no additional configuration.
+These include:
+
+* withkoji.com
+* koji.sh
+* koji.game
+
+You can also use a domain name that you purchase from somewhere like *Google Domains* or *Namecheap*.
+
+== Using a Koji subdomain
+
+. Open the Koji and click the Koji button.
+
+. Go to *Settings > Custom domains*.
+
+. If you have already configured a domain and want to change it, follow the steps for <<_releasing_a_domain>> and then come back to this step.
+If you have not already configured a domain for this Koji, click *New*.
+
+. Select *Subdomain*.
+
+. Click *Next*.
+
+. Select a *root domain*.
+
+. Enter the name that you want to come before the subdomain.
+
+. Click *Save*.
+If the domain name you chose is already taken, "That domain is not available" will be displayed.
+If the domain name is available, the complete domain name will be displayed.
+
+. Click the domain name.
+You should see "Your Koji has been successfully configured to start receiving traffic from your custom domain".
```

Comment:
I'm hesitant to use exact UI text here. As we've seen, it's subject to frequent change. Maybe craft something more generic to explain what the user should see.